test/CI/Travis/apache_vhost

```diff
@@ -0,0 +1,68 @@
+# Configuration file for Apache running on Travis.
```

Comment:
Are you sure, that such complex setup is needed. Here is how the Selenium tests are setup at Mink: https://github.com/minkphp/MinkSelenium2Driver/blob/master/.travis.yml#L27-L30